### PR TITLE
Creation of endpoint to cancel reservation of a room

### PR DESCRIPTION
**Pull request by Benjamin-Chain**
## Summary
- Modified some method in manager layer with unnecessary boolean type
- Created the method to cancel a reservation
- Created the endpoint with params to cancel a reservation

### CODE REVIEW

File: src/main/java/com/teamcubation/librarymanagement/adapter/in/web/RoomController.java [hunk 1]
```diff
@@ -34,6 +34,13 @@ public ResponseEntity<Room> reserveRoom(@RequestParam String address, @RequestPa
         return ResponseEntity.ok(room);
     }
 
+    @PostMapping(path = "/api/rooms/reserve/cancel")
```
Comment (gnievestm) [inline src/main/java/com/teamcubation/librarymanagement/adapter/in/web/RoomController.java]: change this to use a Patch and the same url that is used to reserve.

**Review by gnievestm — CHANGES_REQUESTED** [review]
Fix url please

File: src/main/java/com/teamcubation/librarymanagement/adapter/in/web/RoomController.java (hunk 1 above)
Comment (Benjamin-Chain) [reply src/main/java/com/teamcubation/librarymanagement/adapter/in/web/RoomController.java]: k

**Comment by Benjamin-Chain** [comment]
@gnievestm Done

**Review by gnievestm — APPROVED** [review]

**Pull request merged**